libs/net/dal/Migrations/Initial/Up/PostUp/02-Ingest.sql

```diff
@@ -461,8 +461,10 @@ INSERT INTO public.ingest (
   , (SELECT id FROM public.source WHERE code = 'GLOBE') -- source_id
   , 'GLOBE' -- topic
   , frontPageId -- product_id
-  , '{ "path": "binaryroot",
-      "fileName": "sv-GLB",
+  , '{ "path": "gandm/papers",
```

Comment:
This won't update a DB that has already run this migration, but it does capture that main changes for future installs.